org.lflang/src/org/lflang/TargetConfig.java

```diff
@@ -156,6 +157,12 @@ public class TargetConfig {
      */
     public boolean fastMode = false;
 
+    /**
+     * If fastMode is true, this field records the AST Node that made 
+     * fastMode to be true.
+     */
+    public Element fastModeASTNode = null;
```

Comment:
It looks like this field is set but never read. Is this needed?

Reply:
It's needed in https://github.com/lf-lang/lingua-franca/pull/1413/files#diff-36305ea0c57443b1c6234992d94f5217bb222b410cb31b339b0582d9e96178bcR425 (sole purpose is to provide error message)